core/base/supportedseries_linux_test.go

```diff
@@ -33,7 +33,7 @@ func (s *SupportedSeriesLinuxSuite) TestLatestLts(c *gc.C) {
 		latest, want string
 	}{
 		{"testseries", "testseries"},
-		{"", "jammy"},
+		{"", "noble"},
```

Comment:
I'm a little concerned that this patch has an unexpected side effect.
We wanted "if you have deployed a charm on noble, that you can migrate/upgrade because it is supported"
And we are getting "and now the default for 3.4 is to deploy noble if not otherwise specified"
Is there a way that we can keep "the default LTS for 3.4 is jammy, but if you have deployed something on noble on request, that is ok and upgrades are allowed"

Reply:
I believe this is already the case, see QA steps

```
$ juju deploy ./juju-qa-test_ubuntu-22.04-amd64-arm64_ubuntu-24.04-amd64-arm64.charm
Located local charm "juju-qa-test", revision 0
Deploying "juju-qa-test" from local charm "juju-qa-test", revision 0 on ubuntu@22.04/stable
```